.gitallowed

```diff
@@ -0,0 +1,2 @@
+http:\/\/localhost:8080\/test
```

Comment:
Don't think you need to ignore this, it's just meant for those custom protocol urls where they have 'passwords' in. (Though the addition of these password urls is pre-emptive)